l10n_fr_state/data/res_country_state_data.yml

```diff
@@ -23,173 +24,102 @@
   Creating a res.country.state record
 - 
   !record {model: res.country.state, id: res_country_state_alsace}:
-    code: A
+    code: 44
```

Comment:
Well, in fact, the current code is based on another norme (see description). It could be better to use another name 'geoname_code' (or something similar) to avoid to break current use and keep consistency. Could you add to a little in the description file ? 

kind regard.